### DISCUSSION TITLE
Tree sitter for Mojo

### DISCUSSION THREAD

**Discussion by sa-** [modular/modular#2399] [Ideas]
Is there an implementation for Mojo in tree-sitter? It would open the doors for support in a variety of IDEs - notably Zed which runs really fast and has a lower memory footprint.

**Comment by Sharktheone** [comment]
Unfortunately there is no formal grammar for Mojo currently. There is a [TextMate grammar](https://github.com/modularml/mojo-syntax) though, but from what I know, it can't be used for Treesitter.

I guess updates will be posted in #401
